Cython/Build/Dependencies.py

```diff
@@ -599,7 +599,7 @@ def find_pxd(self, module, filename=None):
     def cimported_files(self, filename):
         if filename[-4:] == '.pyx' and path_exists(filename[:-4] + '.pxd'):
             pxd_list = [filename[:-4] + '.pxd']
-        elif filename[-3:] == '.py'and path_exists(filename[:-3] + '.py'):
+        elif filename[-3:] == '.py'and path_exists(filename[:-3] + '.pxd'):
```

Comment:
```suggestion
        elif filename[-3:] == '.py' and path_exists(filename[:-3] + '.pxd'):
```

... just tidied up a space while we're changing it.